apps/web/components/chat/message-input.tsx

```
@@ -144,16 +146,22 @@ export function MessageInput({ channelName, draft, replyTo, onCancelReply, onSen
   async function handleSend() {
     if ((!content.trim() && files.length === 0) || sending) return
     setSending(true)
+    setSendError(null)
+    setUploadProgress(files.length > 0 ? 0 : null)
     onSent?.()
     try {
-      await onSend(content, files)
+      await onSend(content, files, (percent) => setUploadProgress(percent))
       setContent("")
       onDraftChange("")
       fileUrlCache.current.forEach((url) => URL.revokeObjectURL(url))
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)